config/core/lab-configs.yaml

```diff
@@ -13,12 +13,12 @@ labs:
             - preempt-rt
 
   lab-broonie:
-    lab_type: lava_rest
+    lab_type: lava.lava_xmlrpc
     url: 'https://lava.sirena.org.uk/'
     priority: low
 
   lab-cip:
-    lab_type: lava_rest
+    lab_type: lava.lava_xmlrpc
     url: 'https://lava.ciplatform.org/'
```

Comment:
Does it need `RPC2` in the URL, for `lava.lava_xmlrpc`?

Reply:
Yes it does, fixed.